sdk/keyvault/azure-keyvault-certificates/tests/_async_test_case.py

```diff
@@ -0,0 +1,58 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import os
+
+from azure.keyvault.certificates import ApiVersion
+from azure.keyvault.certificates._shared.client_base import DEFAULT_VERSION
+from devtools_testutils import AzureRecordedTestCase, is_live
+import pytest
+
+
+def get_decorator(**kwargs):
+    """returns a test decorator for test parameterization"""
+    versions = kwargs.pop("api_versions", None) or ApiVersion
+    params = [pytest.param(api_version) for api_version in versions]
+    return params
+
+
+class AsyncCertificatesClientPreparer(AzureRecordedTestCase):
+    def __init__(self, **kwargs) -> None:
+        self.azure_keyvault_url = "https://vaultname.vault.azure.net"
+
+        if is_live():
+            self.azure_keyvault_url = os.environ["AZURE_KEYVAULT_URL"]
+
+        self.is_logging_enabled = kwargs.pop("logging_enable", True)
+        
+        if is_live():
+            os.environ["AZURE_TENANT_ID"] = os.environ["KEYVAULT_TENANT_ID"]
+            os.environ["AZURE_CLIENT_ID"] = os.environ["KEYVAULT_CLIENT_ID"]
+            os.environ["AZURE_CLIENT_SECRET"] = os.environ["KEYVAULT_CLIENT_SECRET"]
+            
+    def __call__(self, fn):
+        async def _preparer(test_class, api_version, **kwargs):
+
+            #self._skip_if_not_configured(api_version)
```

Comment:
We can uncomment this in the sync class as well:
```suggestion
            self._skip_if_not_configured(api_version)
```